docs/application/dotnet/guides/nui/animation-types.md

```diff
@@ -1,127 +1,45 @@
-if (focusData.Name == "halo")
-{
-   keyFrames.Add(focusData.KeyFrameEnd + 0.2f, initSize);
-}
-
-_animation.AnimateBetween(focusData.ImageItem, "Size", keyFrames, Animation.Interpolation.Linear, new AlphaFunction(AlphaFunction.BuiltinFunctions.EaseOutSine));
-```
-
-<a name="2"></a>
 ## Path Animation
 
-The `Path` class can be used to animate the position and orientation of actors. (a three-dimentional parametric curve)
+The `Path` class is three-dimensional parametric curve that is used to animate the position and orientation of actors.
 
-The black points are the points where the 'dali' logo travels. The red points are the control points which express the curvature of the path on the black points.
+The `Path` consists of several points and each adjacent pair of points becomes a segment of the path. Each segment can be curved by adding control points to it.
 
-**Figure: Path animation**
+In the following example, the P1, P2 and P3 form the path where the blue circle travels. The C1 and C2 are the control points for a first segment `[P1-P2]`, where C3 and C4 are for a second segment `[P2-P3]`. The control points express the curvature of the path.
 
-![Path animation](./media/path_animation.png)
+<div style="text-align:center;width:100%;"><img src="./media/path.svg" /></div>
 
-The following code presents the black points:
+To animate, use the following code:
 
-```
+```csharp
 Path path = new Path();
-path.AddPoint(new Vector3( 50.0f, 10.0f, 0.0f ));
-path.AddPoint(new Vector3( 90.0f, 50.0f, 0.0f ));
-path.AddPoint(new Vector3( 10.0f, 90.0f, 0.0f ));
-```
-
-The control points can be added manually using `AddControlPoint`. The `Path` class auto-generates the control points:
 
-```
-path.GenerateControlPoints( 0.25f );
-```
-Here 0.25f represents the curvature of the path you require.
-
-To animate `view1` along this path, use the following function:
-
-```
-_animation.AnimatePath( view1, path, Vector3::ZERO );
-```
+// Path Points
+path.AddPoint(new Position(0, 100));   // P1
+path.AddPoint(new Position(100, 100)); // P2
+path.AddPoint(new Position(200, 100)); // P3
 
-The third parameter is the forward vector (in a local space coordinate system) that is oriented with the path's tangent direction.
-
-Another example:
+// Control Points for a first segment
+path.AddControlPoint(new Position(40, 20)); // C1
+path.AddControlPoint(new Position(60, 20)); // C2
 
+// Control Points for a second segment
+path.AddControlPoint(new Position(140, 180)); // C3
+path.AddControlPoint(new Position(160, 180)); // C4
 ```
-// Black points
-Position position0 = new Position(200.0f, 200.0f, 0.0f);
-Position position1 = new Position(300.0f, 300.0f, 0.0f);
-Position position2 = new Position(400.0f, 400.0f, 0.0f);
 
-Path path = new Path();
-path.AddPoint(position0);
-path.AddPoint(position1);
-path.AddPoint(position2);
+To animate `View` along this path, use the following function:
```

Comment:
Question. In the sample code, `view` is used.
However in this line `View` is described.
Don't we need to use small letter for `View`?

Reply:
`View` is a class name of animation target,
and `view` in the sample code is an example instance of `View` class.

Animation target is always `View` class and its instance can have any name such as `view`, `imageView` and `videoView`, so I used `View` in the description.